src/main/java/org/terasology/dynamicCities/construction/Construction.java

```diff
@@ -563,10 +562,8 @@ public RoadStatus buildRoadParcel(RoadParcel parcel, EntityRef settlement) {
             }
 
             // Check if the region is relevant
-            Region3i region = Region3i.createFromMinMax(
-                    new Vector3i(segment.rect.minX(), vertLimit, segment.rect.minY()),
-                    new Vector3i(segment.rect.maxX(), -1 * vertLimit, segment.rect.maxY())
-            );
+            BlockRegion region = new BlockRegion(segment.rect.minX(), vertLimit, segment.rect.minY(),
+                    segment.rect.maxX(), -1 * vertLimit, segment.rect.maxY());
```

Comment:
Just double-checking here: is `vertLimit` negative, or why do we multiply by -1 to get the max value?

Reply:
no idea